sbi/samplers/mcmc/init_strategy.py

```diff
@@ -45,7 +45,7 @@ def sir(
     $p(\theta_i|\theta_j, x)$ when a `condition` and `dims_to_sample` are passed.
```

Comment:
Add a reference to the doctoring with which users can read up on SIR. or write short explanation of what SIR does.